Raven.Database/Indexing/ReducingExecuter.cs

```diff
@@ -645,15 +648,15 @@ protected override bool ShouldSkipIndex(Index index)
         protected override bool IsIndexStale(IndexStats indexesStat, IStorageActionsAccessor actions, bool isIdle, Reference<bool> onlyFoundIdleWork)
         {
             onlyFoundIdleWork.Value = false;
-            var isReduceStale = actions.Staleness.IsReduceStale(indexesStat.Id);
 
-            if (isReduceStale == false)
+            if (indexesStat.Priority.HasFlag(IndexingPriority.Disabled))
```

Comment:
Just to double-check... This is used only to detect if reducing executer needs to do some work on that index or not? It is not used anywhere alse?

Also, do we have similar logic for IndexingExecuter?

Reply:
It is used only in AbstractIndexingExecuter.
IndexingExecuter has the same logic:
https://github.com/ravendb/ravendb/blob/v3.5/Raven.Database/Indexing/IndexingExecuter.cs#L72